scipy/stats/tests/test_fit.py

```diff
@@ -80,7 +99,7 @@ def test_cont_fit(distname, arg):
         with np.errstate(all='ignore'), suppress_warnings() as sup:
             sup.filter(category=DeprecationWarning, message=".*frechet_")
             rvs = distfn.rvs(size=fit_size, *arg)
```

Comment:
just curious what value of `fit_size` is being used? It could be that increasing that number a bit allows some tests to pass that are currently failing.

Reply:
I'm not sure; I didn't write that. I just worked method of moments into the existing structure. I'll run for the distributions in `mm_failing_fits` with a bigger size and get back to you.

Reply:
`nakagami` passes `MM` with `fit_size = 10000` but causes `chi`, `f` to fail and `truncnorm` to become super slow for `MLE`.

Reply:
> `nakagami` passes `MM` with `fit_size = 10000` but causes `chi`, `f` to fail and `truncnorm` to become super slow for `MLE`. So I don't think I should mess with it right now.

Yeah I agree, there were 2 reasons why I was asking, one was to get the tests to have more coverage by increasing the `fit_size`, the other was more to see if we could ***cut down*** the number of samples somewhat and still have the same coverage, that only makes sense if the current value of `fit_size` is large and the tests take awhile.

Reply:
One nice thing about Method of Moments, I suppose, is that the time taken is barely dependent on the number of samples being fit (since you only need to calculate the sample moments once). But I see how it would be nice to cut down for MLE.